### PR TITLE
Clock App based on new designs - built with Cloudflare workers

### PR DESCRIPTION
**Pull request by iamplugged**
Specs:
https://www.figma.com/file/WcusWuWZDmdHVWgQlieqSo/TV-Device?node-id=1868%3A883

- The app uses local device time. Does not support lat, lon base time timezone derivation.
- Supports 12/24H format based on country locale

Pending
-Test caching

### CODE REVIEW

File: src/index.jsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    headers: {
+      'Cache-Control': 'maxage=43200',
+      'Content-Type': 'text/html; charset=UTF-8'
+    }
+  })
+  
+  return response
+})
+
+//app.get('*', cache({ cacheName: 'default', cacheControl: 'max-age=43200' }))
```
Comment (vpetersson) [inline src/index.jsx]: Is this left in by design?

File: src/index.jsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import { Hono } from 'hono'
+// import { cache } from 'hono/cache'
```
Comment (vpetersson) [inline src/index.jsx]: Is this left in by design?

File: wrangler.toml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+workers_dev = true
+
+[env.stage]
+workers_dev = false
+
+[env.stage.vars]
+ENV = "stage"
+
+[env.production]
+workers_dev = false
```
Comment (vpetersson) [inline wrangler.toml:23]: URL are missing here.

File: wrangler.toml [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+bucket = "./assets"
+
+[dev]
+port = 8888
+
+[env.dev]
+workers_dev = true
+
+[env.stage]
+workers_dev = false
```
Comment (vpetersson) [inline wrangler.toml:16]: URL is missing here.

**Review by vpetersson — CHANGES_REQUESTED** [review]

**Comment by vpetersson** [comment]
The required secrets should now be available.

File: src/index.jsx (hunk 1 above)
Comment (iamplugged) [reply src/index.jsx]: Yes, let's see if we can just use browser default caching. We take a decision on it after testing in stage.

File: wrangler.toml (hunk 3 above)
Comment (iamplugged) [reply wrangler.toml:23]: Updated

File: wrangler.toml (hunk 4 above)
Comment (iamplugged) [reply wrangler.toml:16]: Updated

**Review by vpetersson — APPROVED** [review]

**Pull request merged**